atlasdb-client/src/main/java/com/palantir/atlasdb/util/SlidingWindowMetricsInvocationHandler.java

```diff
@@ -72,10 +71,7 @@ public void onSuccess(@Nullable InvocationContext context, @Nullable Object resu
     public void onFailure(@Nullable InvocationContext context, @Nonnull Throwable cause) {
         markGlobalFailure();
         if (context == null) {
-            log.debug(
-                    "Encountered null metric context likely due to exception in preInvocation: {}",
-                    UnsafeArg.of("cause", cause),
-                    cause);
+            log.debug("Encountered null metric context likely due to exception in preInvocation", cause);
```

Comment:
DRY: Logging the cause as a throwable provides everything that cause.toString does, and more!